lidia/Makefile

```diff
@@ -0,0 +1,28 @@
+GO ?= go
```

Comment:
do we need a separate makefile? can we just test & lint from the main Makefile, same as we currenlty do for ebpf module?